ash/src/vk/enums.rs

```diff
@@ -1054,55 +1054,30 @@ impl ObjectType {
 }
 impl ObjectType {
     pub const UNKNOWN: Self = Self(0);
-    #[doc = "VkInstance"]
```

Comment:
All these `comment=` went away, we have to do a reverse lookup on `objtypeenum` for every type now if we want this comment.

 https://github.com/KhronosGroup/Vulkan-Headers/compare/d2308183f2846a1453ce7ae8c77ea7b5a7424ad2...9efc4a631161eeea11082bbd16d605cca0b7a01e#diff-a852a94832433f254f7e5ff6cf889035be373606623d793dfe4a75e535600cbeR5681